andela/ah-fortem-frontend
# 166980305-user signup

#### Title

Users should be able Signup/Register to Authors Haven
#### Description

To be able to use the application and access authenticated routes, Users should be able to Signup and or Register using Sign up forms on the signup route.




#### Type of change








#### How Has This Been Tested?
- Clone the repo
- Branch to `story/166980302-user-signup`
- Run `npm install` to install the dependencies
- Run `npm run start` to run the app in production
- Navigate to the signup page `/signup`
- Fill in the fields and submit
- You should be able to login to the app after registration
#### Acceptance Criteria: 


- [x] The user should be able to register
- [x] The user should be able to login after registration
- [x] The user should get relevant descriptive error messages

#### Checklist:
- [x] Signup form
- [x] Error messages

#### Screenshots
<img width="1426" alt="Screenshot 2019-07-09 at 13 01 21" src="https://user-images.githubusercontent.com/29132159/60879159-c3a20b80-a249-11e9-966d-64485a72cb44.png">


#### PT stories
[#166980305](https://www.pivotaltracker.com/story/show/166980302)
